src/libponyc/codegen/codegen.c

```diff
@@ -882,7 +882,7 @@ bool codegen_init(pass_opt_t* opt)
   if(opt->features != NULL)
     opt->features = LLVMCreateMessage(opt->features);
   else
-    opt->features = LLVMCreateMessage("");
+    opt->features = LLVMGetHostCPUFeatures();
```

Comment:
This would break cross compiling since it would always set CPU features to those of the host if no CPU features were specified.

I'd suggest instead the following logic:

```C
if(opt->features != NULL)
  opt->features = LLVMCreateMessage(opt->features);
else
  if((opt->cpu == LLVMGetHostCPUName()) && (opt->triple == default_triple_ifdef))
    opt->features = LLVMGetHostCPUFeatures();
  else
    opt->features = LLVMCreateMessage("");
```

I think this would ensure that the host CPU features would only be used only for a native compile for the current host and not if someone was cross compiling.